config/config.go

```diff
@@ -200,6 +202,34 @@ type CustomChainConfig struct {
 	SDKNetwork *sdkConfig.Network `koanf:"sdk_network"`
 }
 
+// ArchiveConfig is information about the nodes for a network.
+type ArchiveConfig struct {
+	// DefaultNode is information about the node to get data from by default.
+	DefaultNode *NodeConfig `koanf:"default"`
+	// ConsensusNode is information about the node to get consensus data from,
+	// instead of the default node.
+	ConsensusNode *NodeConfig `koanf:"consensus"`
+	// RuntimeNodes is the information about the nodes to get runtime data
+	// from, instead of the default node.
+	RuntimeNodes map[common.Runtime]*NodeConfig `koanf:"runtimes"`
+}
+
+func (ac *ArchiveConfig) ResolveConsensus() *NodeConfig {
```

Comment:
Suggestion: `ResolvedConsensusNode()`?

`ResolveD` to make it clearer that it's returning the resolved value, not causing a side effect in the state of `ArchiveConfig`.

`ResolvedConsensusNODE` for symmetry with the public `ConsensusNode` field, and because node is the root of the noun phrase, so it feels weird/misleading to omit it.

Same for `ResolveRuntime` below.

Reply:
ok